.changeset/fix-env-mutation.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ai/core': patch
```

Comment:
there was a bug here (fixed it on main) - for future reference, you can create changesets by running `pnpm changeset` in the project root